### PR TITLE
docs: add documentation around the new `x-codeSamples` spec extension

### PR DESCRIPTION
**Pull request by paulRbr**
We will soon release the support of the specification extension
`x-codeSamples` so this commit adds a help page dedicated to explain
how to use this extension.

Any help will be very welcome on the wording of this 🙏

### CODE REVIEW

**Review by vvenance — COMMENTED** [review]
other than that, LGTM

File: docs/help/specifications-support/openapi-support.md [hunk 1]
```diff
@@ -15,7 +15,14 @@ We do not support `mutualTLS`. To describe a `mutualTLS` authentication method,
 
 JSON Schema provides the possibility to declare a property as read or write only. Read more in [the JSON Schema section of this documentation](/specifications-support/json-schema.md#readonly-and-writeonly-properties).
 
-## Add topics to your documentation
+## Specification extensions
+
+These are additional properties which are not specificed by the OpenAPI specification but useful for you to be able to customize your documentation content. All those properties start with the `x-` naming convention to be identified as “eXternal” from the OpenAPI specification.
```
Comment (jayk-u) [inline docs/help/specifications-support/openapi-support.md:20]:
```suggestion
These are additional properties that are not specified by the OpenAPI specification but are helpful for you to customize your documentation content. All those properties start with the `x-` naming convention to be identified as “eXternal” from the OpenAPI specification.
```

**Review by jayk-u — COMMENTED** [review]
Few suggestions.
A quick reminder that we'll use the gif, even if it will probably be replaced for the V2 (as we need to have a standardized view on them, that's not personal).

**Review by jayk-u — APPROVED** [review]

**Comment by paulRbr** [comment]
@jayk-u don't hate me I made a slight modification to add the information about which programming languages are supported for color syntax, and rephrased the sentences before and after the screen cap.

Do you mind taking a second look?

Thanks :pray:

**Review by jayk-u — APPROVED** [review]

**Pull request merged**